src/cpu/aarch64/matmul/acl_matmul.hpp

```diff
@@ -135,21 +137,35 @@ struct acl_matmul_t : public primitive_t {
                         amp_, src_md_, weights_md_, dst_md_, *desc(), *attr()));
             }
 
+            // We can only fuse sum if it is the first post op and we aren't
+            // transposing dst after
+            if (attr_.post_ops_.contain(primitive_kind::sum, 0) && !amp_.do_transC) {
```

Comment:
Clang format is unhappy about this "long" line.

Reply:
Fixed, sorry, I cherry picked the wrong commit.